static/js/publisher/builds/index.js

```diff
@@ -63,6 +73,10 @@ class Builds extends React.Component {
       .then(res => res.json())
       .then(result => {
         this.setState({
+          triggerBuildLoading:
+            triggerBuildStatus === SUCCESS ? !SUCCESS : triggerBuildStatus,
+          triggerBuildStatus:
+            triggerBuildStatus === SUCCESS ? IDLE : triggerBuildStatus,
```

Comment:
I think the logic for these two needs clarifying - I get console errors. The first return of /builds.json marks it as done. On next fetch of /builds.json I get:
```
Warning: Failed prop type: Invalid prop `isLoading` of type `string` supplied to `TriggerBuild`, expected `boolean`.
    in TriggerBuild (created by Builds)
    in Builds react_devtools_backend.js:6:5119
```

Reply:
was supposed to use `triggerBuildLoading` instead on line 77